tcf_core/settings/base.py

```diff
@@ -1,6 +1,6 @@
-# pylint: disable=line-too-long,import-error,fixme
+# pylint: disable=fixme
 """
-Django settings for tcf_core project.
+Base Django settings for tcf_core project.
 
 Generated by 'django-admin startproject' using Django 3.0.4.
```

Comment:
Not very relevant, but should we remove this? lol

Reply:
Removed in ece568ea729e7072ed9a42f67061f81745158677 and now the file is 11 lines shorter.